mkwheelhouse.py

```diff
@@ -68,7 +68,9 @@ def has_key(self, key):
 
     def generate_url(self, key):
         key = self.get_key(key)
-        return key.generate_url(expires_in=0, query_auth=False)
+        url = key.generate_url(expires_in=0, query_auth=False)
+        # Temporary hack for fixing the links
+        return url.replace('https', 'http').split('?')[0]
```

Comment:
I mean, I don't even know why boto throws in ACL parameters there :) But they just don't work.